README.md

```diff
@@ -4,11 +4,33 @@ This repository contains all python files and (supplementary) figures for the ma
 
 ## Preparations
 
-The study was performed on Linux (Ubuntu 20.10) with Anaconda python 3.8.5. All scripts can be found under `Python_Scripts`, which we henceforth assume to be the working directory (`cd Python_Scripts`).
+The study was performed on Linux (Ubuntu 20.10) with Anaconda python 3.8.5. 
+All scripts can be found under `Python_Scripts`, which we henceforth assume to be the working directory (`cd Python_Scripts`).
+We recommend the installation of a virtual environment using conda:
+
+    conda create -n solverstudy python=3.8
+    conda activate solverstudy
+
 All required dependencies can be installed via
 
     pip install -r ../Software/requirements.txt
 
+In the case of an virtual environment, the requirements must be 
+installed to the specific conda environment, i.e.,
+
+    ~/anaconda/envs/solverstudy/bin/pip install -r ../Software/requirements.txt
```

Comment:
ususally, `which pip` should already point to the correct one once `conda activate solverstudy` was successful. but full path does not hurt.

Reply:
Tried it, didn't work for me... :/

Reply:
hm ... then maybe your conda installation was not properly set up. did you add sth like

```python
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/home/yannik/anaconda3/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/home/yannik/anaconda3/etc/profile.d/conda.sh" ]; then
        . "/home/yannik/anaconda3/etc/profile.d/conda.sh"
    else
        export PATH="/home/yannik/anaconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```
to your .bashrc?

Reply:
Nope, indeed, I didn't...
Good point, thanks!
However: I'd prefer that the correctness of the study relies on things like this to be done manually by the user...

Reply:
This is part of what should be done automatically during the anaconda installation, i.e. that it overrides your default python location. If one does not do that, the python env needs to be set manually every time (also make sure that calls to "python" refer to the correct one throughout then!)